src/Chapter1.hs

```diff
@@ -536,8 +541,15 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | y <= x && x <= z = x
+    | z <= x && x <= y = x
+    | x <= y && y <= z = y
+    | z <= y && y <= x = y
+    | x <= z && z <= y = z
+    | y <= z && z <= x = z
+    | otherwise = 0
```

Comment:
It looks like you can move all these 3 cases under a single `otherwise` and return `z` in here.
The implementation will be a bit shorter :slightly_smiling_face: